mmv1/products/apigee/terraform.yaml

```diff
@@ -218,6 +218,31 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       delete_minutes: 30
     custom_code: !ruby/object:Provider::Terraform::CustomCode
       custom_import: templates/terraform/custom_import/apigee_environment_group_attachment.go.erb
+  EndpointAttachment: !ruby/object:Overrides::Terraform::ResourceOverride
+    autogen_async: true
+    import_format: ["{{org_id}}/endpointAttachments/{{endpoint_attachment_id}}", "{{org_id}}/{{endpoint_attachment_id}}"]
+    delete_url: '{{org_id}}/endpointAttachments/{{endpoint_attachment_id}}'
+    skip_sweeper: true
+    examples:
+    - !ruby/object:Provider::Terraform::Examples
+      name: "apigee_endpoint_attachment_basic"
+      skip_test: true
+    - !ruby/object:Provider::Terraform::Examples
+      # This is a more verbose version of the above that creates all
+      # the resources needed for the acceptance test.
+      name: "apigee_endpoint_attachment_basic_test"
+      primary_resource_id: "apigee_endpoint_attachment"
+      test_env_vars:
```

Comment:
will also need a `vars` block that defines variable names for each resource name. See https://github.com/GoogleCloudPlatform/magic-modules/blob/master/mmv1/products/pubsublite/terraform.yaml#L51